djangocms_multisite/middleware.py

```diff
@@ -9,20 +9,29 @@
 
 class CMSMultiSiteMiddleware(object):
     def process_request(self, request):
+        MULTISITE_CMS_URLS = getattr(settings, 'MULTISITE_CMS_URLS', {})
```

Comment:
I'd prefer to have these 3 constants at either class or even module level to decrease the amount of those lookups